src/models/like.c

```diff
@@ -44,6 +44,33 @@ Like *likeCreate(sqlite3 *DB, int accountId, int authorId, int postId)
     return like;
 }
 
+Like *likeDelete(sqlite3 *DB, int accountId, int authorId, int postId)
+{
+    int rc, t;
+    Like *like = NULL;
+    sqlite3_stmt *statement;
+
+    t  = time(NULL);
+    rc = sqlite3_prepare_v2(DB,
+			    "DELETE"
+			    " FROM likes"
+			    " WHERE account = ?"
+			    "   AND post = ?",
+			    -1, &statement, NULL);
+
+    if (rc != SQLITE_OK) return NULL;
+    if (sqlite3_bind_int(statement, 1, accountId) != SQLITE_OK) goto fail;
+    if (sqlite3_bind_int(statement, 2, postId) != SQLITE_OK) goto fail;
```

Comment:
Teacher, the likeCreate function( in like.c line19 ) also write like this

`if (sqlite3_bind_int(statement, 1, t) != SQLITE_OK) goto fail;`
`if (sqlite3_bind_int(statement, 2, accountId) != SQLITE_OK) goto fail;`
`if (sqlite3_bind_int(statement, 3, authorId) != SQLITE_OK) goto fail;`
`if (sqlite3_bind_int(statement, 4, postId) != SQLITE_OK) goto fail;`

Could I rewrite it ?